### PR TITLE
Name and URL Update

### PR DESCRIPTION
**Pull request by solanacryptodev**
I added a new field in the Drop creation and edit section for 'NFT Name' which corresponds with the name value in the JSON output file. This changes it from being identical to the Symbol which is how it was set previously. I also edited the External URL so that it doesn't append the NFT data to it and only prints the website url by itself.

### CODE REVIEW

File: functions/models/candymachine.ts [hunk 1]
```diff
@@ -112,20 +112,16 @@ export namespace CandyMachine {
...
-        "/" +
-        collection.symbol.toLowerCase() +
-        "/" +
-        humanReadableOrderNumber,
+        project.url,
       image: orderNumber + ".png",
       attributes: attributes,
       collection: {
-        name: collection.name,
+        name: collection.collectionName,
```
Comment (cosimo-rip) [inline functions/models/candymachine.ts:124]: could we keep `collection.name` as the name of the collections and make the new property `nftName`?

File: pages/projects/[projectId]/collections/[collectionId]/edit.tsx [hunk 2]
```diff
@@ -91,21 +93,41 @@ export default function EditPage(props: Props) {
...
+                      <input
+                          type="text"
+                          name="name"
+                          id="name"
+                          placeholder="Name of NFT"
+                          defaultValue={collection.name}
+                          className="mt-1 block w-full shadow-sm sm:text-sm rounded-md"
+                      />
+                      <p className="mt-2 text-xs text-gray-500">
+                        This is the name of the NFT that will be generated in the JSON file
```
Comment (cosimo-rip) [inline pages/projects/[projectId]/collections/[collectionId]/edit.tsx:127]: can we note in this description that the name will have the index number appended?

"This is the name that will be shown on the NFT. The index number will be appended, i.e. "NAME 542".

**Review by cosimo-rip — CHANGES_REQUESTED** [review]
Thanks so much for the contribution. Left a couple small notes.

Also are the yarn.lock and package-lock.json changes necessary? if not you could leave those out.

File: functions/models/candymachine.ts (hunk 1 above)
Comment (solanacryptodev) [reply functions/models/candymachine.ts:124]: Will do tomorrow!

File: pages/projects/[projectId]/collections/[collectionId]/edit.tsx (hunk 2 above)
Comment (solanacryptodev) [reply pages/projects/[projectId]/collections/[collectionId]/edit.tsx:127]: Will do tomorrow!

**Pull request closed without merging**